control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -67,6 +67,18 @@ func ReconcileRole(role *rbacv1.Role, ownerRef config.OwnerRef) error {
 				"watch",
 			},
 		},
+		{
+			APIGroups: []string{hyperv1.GroupVersion.Group},
+			Resources: []string{
+				"hostedcontrolplanes/finalizers",
```

Comment:
The reason this is needed is that the hcco is setting the hcp as the owner of a configmap it reconciles. Without having finalizer access, the API server balks.

Reply:
Ah ok, probably because it is possible to block the owner deletion through an ownerReference. Definitely an ocp-specific thing and to me, referencing a non-existent api will always look wrong, maybe add a godoc explaining this?

Reply:
Will adda godoc. However, I don't think it's an openshift-specific thing:
https://github.com/FairwindsOps/rbac-manager/issues/180

Reply:
Nevermind that *is* openshift :)

Reply:
Very sure its OCP-specific, that issue you linked happens on OCP and the response is "I don't have an openshift cluster to test, but you could try..." which seems to imply it doesn't happen on upstream kube